src/lib.rs

```diff
@@ -707,7 +723,6 @@ impl Ulid {
     /// # Ok::<(), rusty_ulid::DecodingError>(())
     /// ```
     #[cfg(feature = "chrono")]
-    #[must_use]
```

Comment:
Why was this removed?

Reply:
Not super sure how that got removed, re-added.